spec/models/form/sales/property/questions/property_number_of_bedrooms_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe Form::Sales::Questions::PropertyNumberOfBedrooms, type: :model do
```

Comment:
This should be `RSpec.describe Form::Sales::Property::Questions::PropertyNumberOfBedrooms`